src/ray/raylet/local_task_manager.cc

```diff
@@ -999,13 +999,10 @@ ResourceSet LocalTaskManager::CalcNormalTaskResources() const {
     }
 
     if (auto allocated_instances = worker->GetAllocatedInstances()) {
-      auto resource_request = allocated_instances->ToResourceRequest();
-      for (auto entry : resource_request.ToMap()) {
-        total_normal_task_resources.emplace(entry.first.Binary(), entry.second);
-      }
+      total_normal_task_resources += allocated_instances->ToResourceRequest();
```

Comment:
The semantics of this modification is different from that before. Was there a problem before?

Reply:
It should be += operation here because we want to sum up all normal tasks' leased workers. It was wrongly changed at Hao's PR (https://github.com/ray-project/ray/pull/22854/files). @raulchen @wumuzi520